cli.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+print("****************")
+print("TimeTracker CLI")
+print("****************")
+
+import os
+
+from flask import json
+from time_tracker_api import create_app
+from flask_script import Manager
+from time_tracker_api.api import api
+
+app = create_app()
+cli_manager = Manager(app)
+
+
+@cli_manager.command
+@cli_manager.option('-f', '--filename', help='Path of the swagger file. By default swagger.json')
+def gen_swagger_json(filename='swagger.json'):
+    """ Exports swagger specifications in json format """
+    schema_json_data = json.dumps(api.__schema__)
+    save_data(schema_json_data, filename)
+
+
+@cli_manager.command
+@cli_manager.option('-f', '--filename', dest='filename', help='Path of the postman collection file.'
+                                         'By default it is timetracker-api-postman-collection.json')
+def gen_postman_collection(filename='timetracker-api-postman-collection.json'):
+    """ Generates a Postman collection for the API """
+    data = api.as_postman(urlvars=False, swagger=True)
+    postman_collection_json_data = json.dumps(data)
+    parsed_json = postman_collection_json_data.replace("http://localhost", '{{timetracker_api_host}}')
```

Comment:
It would be great if we can send `http://localhost` as a parameter to the `cli.py` file. I think by default it can be localhost.

Reply:
By default it is actually localhost:5000 for development and localhost:8000 when it is using a wsgi server, like gunicorn. The idea behind setting a postman env variable is that you can set that env variable to whatever you want based on the environment you will be working at.
Nonetheless, I will create a param, so you can override the default param which by default will be localhost and everyone is happy :)